README.md

```diff
@@ -6,22 +6,23 @@ LINE Things compatible device example firmware and LIFF application code are inc
 
 ## Getting Started
 
-[LINE Things](https://developers.line.biz/ja/docs/line-things/) is a platform that connects Bots and Web services together with IoT devices that are Bluetooth LE enabled. 
+[LINE Things](https://developers.line.biz/ja/docs/line-things/) is a platform that connects Bots and Web services together with IoT devices that are Bluetooth LE enabled.
 
 To start developing for LINE Things using the example code and the sample firmware, the following steps are necessary.
 
 ### Installing the Firmware
 
 To start development using the example firmwares, you will need a compatible Bluetooth LE enabled development board.
-Currently LINE Things Starter firmwares supports the following 3 development boards
+Currently LINE Things Starter firmwares supports the following 6 development boards
 
+- LINE Things dev board
```

Comment:
I think it would be nice to link this to a blog post or to a page where users can know more about the hardware?

Reply:
I added that.